.github/workflows/deploy.yml

```diff
@@ -32,10 +32,11 @@ jobs:
           sudo apt-get install krb5-multidev
       - name: Verify latest centos image is present
         run: |
-          docker image ls | grep centos
-          docker pull centos:7
-          docker pull centos/s2i-base-centos7
-          docker image ls | grep centos
+          if ! docker image ls | grep -q centos; then
```

Comment:
I'd suggest not doing the if here -- there could be a centos:7 image present somehow and it might not be the latest one, so always running the pull is probably a good idea.

Reply:
I agree, I'll remove it.